reference/settings.v3.yml

```diff
@@ -924,6 +924,21 @@ components:
         store_country:
           type: string
           default: United States
+        shopper_language_selection_method:
+          type: string
+          enum:
+            - browser
+            - default_shopper_language
+          example: browser
+          default: browser
+          description: |-
+            Determines whether to display the storefront content in the shopper's browser language or the shopper's selected default language.
```

Comment:
> Determines whether to display the storefront content in the shopper's browser language(s) or the selected single shopper's default language.

I think it's good to callout here that with shopper browser language multiple languages are supported, but with default shopper language – it's gonna be single selected language